src/js/utils/JestUtil.js

```diff
@@ -51,6 +51,60 @@ const JestUtil = {
       return true;
     }
   },
+  /**
+   * Generates a callback function to a filter() call that will
+   * keep only DOMElements matching the given tag name(s)
+   *
+   * @example <caption>How to use filterByTagName</caption>
+   * const TestUtils = require('react-addons-test-utils');
+   *
+   * var componentInstance = TestUtils.renderIntoDocument(
+   *     <MyTableComponent />
+   *   );
+   *
+   * var nameTDs = TestUtils.scryRenderedDOMComponentsWithClass(
+   *       componentInstance, 'table-column-name')
+   *       .filter(filterByTagName('TD'));
+   *
+   * @param {array|string} tag - One or mor tagNames to match
+   * @returns {function} Returns a callback function to be passed on .filter()
+   */
+  filterByTagName(tag) {
+    if (!Array.isArray(tag)) {
+      tag = [tag];
+    }
+
+    return function (element) {
+      return tag.reduce(function (isAccepted, tagName) {
```

Comment:
I think you can use `.some` for this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some